numpy/__init__.pyi

```diff
@@ -2908,9 +2908,9 @@ class floating(inexact[_NBit1]):
     ) -> float: ...
     def tolist(self) -> float: ...
     def is_integer(self) -> bool: ...
-    def hex(self: float64) -> str: ...
+    def hex(self: _FloatType) -> str: ...
     @classmethod
-    def fromhex(cls: type[float64], string: str, /) -> float64: ...
+    def fromhex(cls: type[_FloatType], string: str, /) -> _FloatType: ...
```

Comment:
@BvB93 , I added for `hex` and `fromhex` as I added both functions. This is correct?